src/main/java/org/fundacion/jala/converter/view/compiler/ProjectTab.java

```diff
@@ -79,10 +82,53 @@ public boolean removeTap(String title) {
     }
 
     /**
-     * It represents what CodeTextArea is selected
-     * @return a CodeTextArea
+     * It represents what CodeTextArea is selected.
+     *
+     * @return a CodeTextArea of a selected tab.
      */
     public CodeTextArea getSelectedPane() {
         return (CodeTextArea) getSelectedComponent();
     }
+
+    /**
+     * Creates a title made up of a name and a close button.
+     *
+     * @param title is text that shows as a title.
+     * @return a JPanel with complete title.
+     */
+    public JPanel makeTabTitle(final String title) {
+        CloseButton tabButton = new CloseButton();
+        tabButton.setPreferredSize(new Dimension(20, 20));
+        tabButton.addActionListener(e1 -> {
+            removeTap(title);
+            setSelectedIndex(getTabCount() - 2);
+        });
+        JPanel pnl = new JPanel();
+        pnl.setLayout(new FlowLayout());
+        pnl.setOpaque(false);
+        JLabel label = new JLabel(title);
+        label.setFont(new Font("Barlow", 0, 11));
```

Comment:
Perhaps, instead of 0 value you could use Font.PLAIN to avoid magic numbers or to made code more readable, just a suggestion.